<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Dialog{
background-color:rgb(245, 245, 245);}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 20pt &quot;Yu Mincho Light&quot;;color:rgb(0, 0, 0);
font: 25 24pt &quot;Yu Mincho Light&quot;;
font: 25 24pt &quot;Yu Mincho Light&quot;;</string>
   </property>
   <property name="text">
    <string>Hello</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>651</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 10pt &quot;Yu Mincho Light&quot;;color:rgb(0,0,0);
font: 25 18pt &quot;Yu Mincho Light&quot;;</string>
   </property>
   <property name="text">
    <string>Welcome to the adaptive learning intelligence software (A.L.I).</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
font: 25 12pt &quot;Yu Mincho Light&quot;;
background-color:rgb(0,0,0);color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>NEXT</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 24pt &quot;Yu Mincho Light&quot;;color:rgb(0,0,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
